src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy;
```

Comment:
Move the entire implementation from the `strategy` package to the `impl` package
```suggestion
package core.basesyntax.strategy.impl;
```